Cargador.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Cargador.h"
+
+    /// METODOS PRIVADOS ///
+//Sobre carga de metodos cargarDatos
+
+void Cargador::cargarDatos(ABB<Diccionario*> &arbolAeropuertos ,string ruta)
+{
+    ifstream archivo;
+    archivo.open(ruta.c_str(),ios::in);
+
+    try:
+        if(archivo.fail()): throw(Excepcion(EXCEPCION_ABB));
+        cargadorDeABB(arbolAeropuertos,archivo);
+        cout << "\n\tSe han cargado los Aeropuertos en el arbol correctamente\n";
+
+    catch(Excepcion &e):
+        cout << e.what() << endl;
```

Comment:
Idem correcion anterior sobre los bloques `try/catch`, siempre llevan llaves
```suggestion
    try {
        if (archivo.fail())
            throw (Excepcion(EXCEPCION_ABB));
        cargadorDeABB(arbolAeropuertos, archivo);
        cout << "\n\tSe han cargado los Aeropuertos en el arbol correctamente\n";
    }
    catch(Excepcion &e) {
        cout << e.what() << endl;
```